creational/factory/Clothes.mjs

```diff
@@ -0,0 +1,16 @@
+export class Clothes {
+  constructor() {
+    this.type = "Clothes";
```

Comment:
Move the initialization of this values to a factory method. Not a part of the class itself